partner-api.yaml

```diff
@@ -506,12 +513,91 @@ paths:
           description: Forbidden
         '404':
           description: Not Found
-
+  '/partner/{partnerId}/zugang':
+    parameters:
+      - schema:
+          type: string
+        name: partnerId
+        in: path
+        required: true
+        description: ID des Partners
+    get:
+      summary: Your GET endpoint
+      tags: []
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ZugangPerson'
+        '403':
+          description: Forbidden - kein Zugriff
+        '404':
+          description: Not Found - Partner nicht gefunden
+      operationId: get-partner-partnerId-zugang
+      description: Liefert die Zugangsdaten eines Partners. Bei einer Person werden die Benutzer-Zugangsdaten und bei einer Organisation die Daten des Identity Provider geliefert.
+    post:
+      summary: Zugang anlegen
+      tags:
+        - Partner
+      operationId: post-partner-partnerId-zugang
+      responses:
+        '200':
+          description: OK - Zugang angelegt
+        '400':
+          description: Bad Request - Anfrage ungültig/konnte nicht verstanden werden
+        '406':
+          description: Not Acceptable
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+              examples:
+                usernameNotAllowed:
+                  value:
+                    message: Username not allowed
```

Comment:
Was genau unterscheidet die Felder `message` und `statusMessage`. Offenbar ist `message` die Kurzform und `statusMessage` ausführlich, aber brauchen wir das wirklich? Und warum steht eigentlich nicht in `message` die ausführliche Meldung und in `statusMessage` (analog zu `statusCode`) die Kurzform?